bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -57,20 +60,15 @@ public OpenGarageHandler(Thing thing) {
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
         try {
-            logger.warn("Received command {} for thing '{}' on channel {}", command, thing.getUID().getAsString(),
-                    channelUID.getId());
-            switch (channelUID.getId()) {
-                case OpenGarageBindingConstants.CHANNEL_OG_STATUS:
-                    if (command instanceof OnOffType) {
-                        changeStatus(((OnOffType) command).equals(OnOffType.ON));
-                        return;
-                    }
-                    break;
-                default:
+            if (channelUID.getId().equals(OpenGarageBindingConstants.CHANNEL_OG_STATUS) || channelUID.getId().equals(OpenGarageBindingConstants.CHANNEL_OG_STATUS_ROLLERSHUTTER)) {
+                if (command.equals(OnOffType.ON) || command.equals(UpDownType.UP)) {
+                    changeStatus("open");
```

Comment:
Can you make these status values enums instead of strings?

Reply:
Done. Hopefully it's ok.